Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -1677,16 +1680,36 @@ void EmbeddedFrag::flipAboutBond(unsigned int bondId, bool flipEnd) {
   endSideAids.clear();
   _recurseAtomOneSide(endAid, begAid, dp_mol, endSideAids);
 
-  // noew we have the molecule split into two groups of atoms
+  // look for fixed atoms in the fragment:
+  unsigned int nEndAtomsFixed = 0;
```

Comment:
The loop above that checks to see if the indices are "allIn".  That's the variable name.  This is more a curiosity than anything else.